autorest/codegen/models/operation.py

```diff
@@ -301,9 +301,7 @@ def imports(
             file_import.merge(self.parameters.body_parameter.type.imports(**kwargs))
 
         # Exceptions
-        if self.abstract:
-            file_import.add_import("abc", ImportType.STDLIB)
-        else:
+        if not self.abstract:
```

Comment:
nit: i don't think we want any imports for abstract, so we could just add a line that is `if self.abstract: return file_import`, so we don't have to keep checking abstract later down the pipeline